client/reader/lib/header-section/index.tsx

```diff
@@ -1,16 +1,22 @@
-import { translate } from 'i18n-calypso';
+import { useTranslate } from 'i18n-calypso';
 import React from 'react';
 
-const renderHeaderSection = () => (
-	<div className="logged-out-reader-header">
-		<h2>
-			{
-				// translators: The title of the reader tag page
-				translate( 'WordPress Reader' )
-			}
-		</h2>
-		<h1>{ translate( 'Enjoy millions of blogs at your fingertips.' ) }</h1>
-	</div>
-);
+const ReaderHeaderSection = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="logged-out-reader-header">
+			<h2>
+				{
+					// translators: The title of the reader tag page
+					translate( 'WordPress Reader' )
+				}
+			</h2>
+			<h1>{ translate( 'Enjoy millions of blogs at your fingertips.' ) }</h1>
```

Comment:
Pre-existing, but I found it an odd semantic decision to have the description/subtitle in a `h1` and the title in a `h2` 🤔

Reply:
Yeah, that does feel unusual. Still, I could also get behind the reasoning that WordPress Reader is more like a "branding" and not necessarily the best description of the page itself.

I can check on this within MarTech to see what's most appropriate! We can always follow up on this.

Reply:
Yep, definitely not a blocker for this PR 👍